query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/ServicesQueries.java

```diff
@@ -53,6 +53,56 @@ static QueryRequest buildQuery1() {
             .build());
 
 
+    builder.addGroupBy(
+        Expression.newBuilder().setColumnIdentifier(serviceId));
+    builder.addGroupBy(
+        Expression.newBuilder().setColumnIdentifier(serviceName));
+    Function serviceDurationPercentileFunc = Function.newBuilder().addArguments(
+        Expression.newBuilder().setColumnIdentifier(ColumnIdentifier.newBuilder().setColumnName("SERVICE.duration").build()).build()).setFunctionName("PERCENTILE99")
+        .build();
+    OrderByExpression orderByExpression = OrderByExpression.newBuilder().setOrder(SortOrder.DESC).setExpression(
+        Expression.newBuilder().setFunction(serviceDurationPercentileFunc).build()).build();
+    builder.addOrderBy(orderByExpression);
+    builder.setLimit(10000);
+
+    return builder.build();
+  }
+
+  static QueryRequest buildQuery2() {
+    Builder builder = QueryRequest.newBuilder();
+    ColumnIdentifier serviceId = ColumnIdentifier.newBuilder().setColumnName("SERVICE.id").build();
+    ColumnIdentifier serviceDuration = ColumnIdentifier.newBuilder().setColumnName("SERVICE.duration").build();
+    ColumnIdentifier serviceName = ColumnIdentifier.newBuilder().setColumnName("SERVICE.name").setAlias("SERVICE.name").build();
+    Function serviceIdAvgRateFunction = Function.newBuilder().addArguments(
```

Comment:
As avg_rate and sum are on duration metrics, shall we rename this as `durationAvgRateFunction` and `durationSumFunction`?